### ISSUE TITLE
i18n panels not fully translated

### ISSUE THREAD

**Issue by jgranduel** [dbeaver/dbeaver#9826]
#### System information: 
- Windows (not tested on other platforms but might be the same)
- 7.2
- Additional extensions

#### Connection specification:
- PostgreSQL but not relevant

#### Describe the problem you're observing:
In French versions, many terms remain in English. I can not find in which bundles terms are defined

examples hereafter (English terms are highlighted)
![image](https://user-images.githubusercontent.com/586675/93754582-466e7680-fc02-11ea-8d84-62b21295e332.png)

Can help if needed.
Thanks

**Comment by kseniiaguzeeva** [comment]
Thank you for the report.

**Comment by LonwoLonwo** [comment]
If you want to help, here is [localization page](https://github.com/dbeaver/dbeaver/wiki/Localization)
Or you can write the original phrases + translation here, and we will carefully transfer it to the beaver

**Comment by jgranduel** [comment]
Thank you. I will try to not be lost in translation!

**Comment by akilovich** [comment]
Hey @jgranduel. It seems like you lost in the translation process indeed. Anyway, we still suggest you write down below your translations in form 

`english_text = french_text`

and I will transfer it to DBeaver as soon as I can. None of the people in our team speaks French, and we definetely don't want to use translators here, so it is an example where the help from our community is really helpful. Thanks.

**Comment by hamdanimed** [comment]
Search = Recherche
Connection view = Vue de connexion
Compare/Migrate = Comparer/Migrer
Copy SQL as source code = Copier SQL comme code source
Editors = Editeurs
Editor = Editeur
Templates = Modèles
Grid = grille
Plain text = Text brut
Timeout des opération longues = Timeout des opérations longues
Save/restore active schema = Enregistrer/Restaurer le schéma actif
Miscellaneous = Divers
Mark occurance of word under cursor = Marquer l'occurrence du mot sous le curseur
Mark occurrences of the selected word = Marquer les occurrences du mot sélectionné

**Comment by flambchap** [comment]
Thank you @hamdanimed

**Comment by KawtharAlakri** [comment]
Hey @n5a5, I would like to work on this but I need some help with running locally for development, if you can help me with that.

**Comment by E1izabeth** [comment]
@KawtharAlakri 
Here is the wiki about it https://github.com/dbeaver/dbeaver/wiki/Localization

**Comment by KawtharAlakri** [comment]
Thanks @E1izabeth but I can't see the imported launch configuration in the debug configuration console 
![image](https://github.com/user-attachments/assets/52c963cf-d3bc-4710-9de8-a9554c354eb0)

It looks differently on my machine: 
![image](https://github.com/user-attachments/assets/a6783d53-2001-4cf9-84d2-4b13875c812e)

**Comment by E1izabeth** [comment]
Do you use Eclipse RCP 2024-06?
https://www.eclipse.org/downloads/packages/release/2024-06/r/eclipse-ide-rcp-and-rap-developers

**Comment by KawtharAlakri** [comment]
oh thanks @E1izabeth, I was in the wrong version. but now when i try to run debug it says: 
![image](https://github.com/user-attachments/assets/e21009c6-467a-4841-afcb-5b3e14bfab0c)

**Comment by E1izabeth** [comment]
@KawtharAlakri please try the following steps https://github.com/dbeaver/dbeaver/wiki/Develop-in-Eclipse#include-plugins-in-the-debug-info

**Comment by KawtharAlakri** [comment]
Unfortunately, this doesn't solve the issue. I noticed that some warning was regarding the execution environment, so I changed it to JSE-17 instead of 11 (the default within run launch configuration) and I the list has minimized, but still missing dependencies even if i select all plugins not even just the required. 
![image](https://github.com/user-attachments/assets/01698dcc-1028-4e40-8827-4ab006f044dd)

sample of missing plugins: 
![image](https://github.com/user-attachments/assets/068dcfbd-803f-445d-bbf2-13aec437493e)

can I add them from anywhere else ?

**Comment by Destrolaric** [comment]
@KawtharAlakri You need to download dependencies https://github.com/dbeaver/dbeaver/wiki/Develop-in-Eclipse#:~:text=features%2C%20and%20products.-,Opening%20workspace%20in%20Eclipse,-You%20will%20need do 1-5 steps to do so.

**Comment by KawtharAlakri** [comment]
@Destrolaric I already did that, but I just noticed that the dependencies that appear to me aren't the same to the attached screenshot the wiki
Documentation:
![image](https://github.com/user-attachments/assets/e1eb2f22-06b4-4572-a2bd-48c2a54399b4)

Mine: 
![image](https://github.com/user-attachments/assets/e4472fcb-27de-417a-aba8-38c157854b39)

**Comment by KawtharAlakri** [comment]
Thanks @Destrolaric @E1izabeth for the suggestions, the missing plugins and dependencies issue is fixed by downgrading to eclipse version 23-12 instead of latest (24-06). 

But now I have errors within the code itself as: 
![image](https://github.com/user-attachments/assets/55098a26-ff87-4211-a6e7-8a0eef13b7c1)
![image](https://github.com/user-attachments/assets/b28caf8e-aae8-408d-add8-c53e844a0941)

I did everything exactly same as the instructions, except for the execution environment in the launch configuration to JavaSE-17 instead of 11 to solve the dependencies issue. 
![image](https://github.com/user-attachments/assets/8067967b-afdc-4b2d-8012-41914867585d)

**Comment by E1izabeth** [comment]
Have you downloaded and added to eclipse [dbeaver-common repository](https://github.com/dbeaver/dbeaver-common)?

**Comment by KawtharAlakri** [comment]
@E1izabeth Yes I have imported to eclipse the directory that has both cloned repos (dbeaver and dbeaver-common) and selected everything: 
![image](https://github.com/user-attachments/assets/e55ffc0a-cb34-4577-ac17-5b9cde75affd)

**Comment by E1izabeth** [comment]
Could you try steps 8-9 from here https://github.com/dbeaver/dbeaver/wiki/Develop-in-Eclipse ?

**Comment by KawtharAlakri** [comment]
Thanks @E1izabeth for the help, I got it fixed by deleting the two projects and re-import each one separately.
